pkg/service/one2onerestore/service.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2025 ScyllaDB
+
+package one2onerestore
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/pkg/errors"
+	"github.com/scylladb/go-log"
+	"github.com/scylladb/gocqlx/v2"
+
+	"github.com/scylladb/scylla-manager/v3/pkg/scyllaclient"
+	"github.com/scylladb/scylla-manager/v3/pkg/service/cluster"
+	"github.com/scylladb/scylla-manager/v3/pkg/service/configcache"
+	"github.com/scylladb/scylla-manager/v3/pkg/util/uuid"
+)
+
+// Service for the 1-2-1 restore.
```

Comment:
nit: let's keep the same naming convention (1-1-restore or one2onerestore).
I guess that we prefer the 1-1-restore and use it when dashes are not a problem.